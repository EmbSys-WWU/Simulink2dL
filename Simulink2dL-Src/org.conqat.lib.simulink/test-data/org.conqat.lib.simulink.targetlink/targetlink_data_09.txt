struct('codefile','S_Regression_TL.c','prodcode',1,'dt','0.01','logdata',struct('maxnumvals',{inf,inf},'plotchannels',{1,1},'loggingmode',{1,1},'channels',{1,1}),'output',struct('variable',{'',''},'description',{'',''},'name',{'slope','lse'},'varname',{'slope','lse'},'width',{1,1},'address',{'',''},'max',{NaN,NaN},'min',{NaN,NaN},'class',{'default','default'},'type',{'Int16','Int32'},'scaling',{'',''},'lsb',{0.001,0.001},'offset',{0,0},'arb',{1,1},'freeze',{1,1},'unit',{'',''}),'input',struct('variable',{'','',''},'description',{'','',''},'name',{'buf','M','T'},'varname',{'buf','M','T'},'width',{32,1,1},'address',{'','',''},'max',{NaN*zeros(1,32),NaN,NaN},'min',{NaN*zeros(1,32),NaN,NaN},'class',{'default','default','default'},'type',{'Int16','UInt8','UInt8'},'scaling',{'','',''},'lsb',{0.001*ones(1,32),1,0.01},'offset',{zeros(1,32),0,0},'arb',{ones(1,32),0,1},'freeze',{1,1,1},'unit',{'','',''},'feedthrough',{1,1,1}),'work',struct('variable','','description','','name','sum','varname','sum','width',1,'address','','max',NaN,'min',NaN,'class','default','type','Int32','scaling','','lsb',0.001,'offset',0,'arb',1,'freeze',1,'unit',''),'scalingvalid',0,'date',733651.60012752318)