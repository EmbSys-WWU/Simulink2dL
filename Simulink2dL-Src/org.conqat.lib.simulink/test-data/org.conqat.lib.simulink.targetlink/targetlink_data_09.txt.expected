struct (
  'input' -> struct (
    'address' -> '{,,}'
    'arb' -> '{ones(1,32),0,1}'
    'class' -> '{default,default,default}'
    'description' -> '{,,}'
    'feedthrough' -> '{1,1,1}'
    'freeze' -> '{1,1,1}'
    'lsb' -> '{0.001*ones(1,32),1,0.01}'
    'max' -> '{NaN*zeros(1,32),NaN,NaN}'
    'min' -> '{NaN*zeros(1,32),NaN,NaN}'
    'name' -> '{buf,M,T}'
    'offset' -> '{zeros(1,32),0,0}'
    'scaling' -> '{,,}'
    'type' -> '{Int16,UInt8,UInt8}'
    'unit' -> '{,,}'
    'variable' -> '{,,}'
    'varname' -> '{buf,M,T}'
    'width' -> '{32,1,1}'
  )
  'logdata' -> struct (
    'channels' -> '{1,1}'
    'loggingmode' -> '{1,1}'
    'maxnumvals' -> '{inf,inf}'
    'plotchannels' -> '{1,1}'
  )
  'output' -> struct (
    'address' -> '{,}'
    'arb' -> '{1,1}'
    'class' -> '{default,default}'
    'description' -> '{,}'
    'freeze' -> '{1,1}'
    'lsb' -> '{0.001,0.001}'
    'max' -> '{NaN,NaN}'
    'min' -> '{NaN,NaN}'
    'name' -> '{slope,lse}'
    'offset' -> '{0,0}'
    'scaling' -> '{,}'
    'type' -> '{Int16,Int32}'
    'unit' -> '{,}'
    'variable' -> '{,}'
    'varname' -> '{slope,lse}'
    'width' -> '{1,1}'
  )
  'work' -> struct (
    'address' -> ''
    'arb' -> '1'
    'class' -> 'default'
    'description' -> ''
    'freeze' -> '1'
    'lsb' -> '0.001'
    'max' -> 'NaN'
    'min' -> 'NaN'
    'name' -> 'sum'
    'offset' -> '0'
    'scaling' -> ''
    'type' -> 'Int32'
    'unit' -> ''
    'variable' -> ''
    'varname' -> 'sum'
    'width' -> '1'
  )
  'codefile' -> 'S_Regression_TL.c'
  'date' -> '733651.60012752318'
  'dt' -> '0.01'
  'prodcode' -> '1'
  'scalingvalid' -> '0'
)